components/service/glean/docs/metrics/adding-new-metrics.md

```diff
@@ -80,15 +80,28 @@ Views.loginOpened...
 There are different metrics to choose from, depending on what you want to achieve:
 
 * [Events](#Events): Records events e.g. individual occurences of user actions, say every time a view was open and from where.
-* [Booleans](#Booleans): Records a single truth value, for example "is a11y enabled?"
+* [Booleans](#Booleans): Records a single truth value, for example "is a11y enabled?".
+* [Labeled booleans](#Labeled-booleans): Records multiple truth values, for example which URI schemes were used or not - `"http"`, `"https"`, `"about"`.
 * [Strings](#Strings): Records a single Unicode string value, for example the name of the OS.
-* [String Lists](#String-lists): Records a list of Unicode string values, for example the list of enabled search engines.
+* [Labeled strings](#Labeled-strings): Records multiple Unicode string values. **TODO: what's the use-case here? Should we block these for now?**
+* [String Lists](#String-lists): Records a list of Unicode string values, for example the list of enabled search engines **TODO - i can't think of a good example here**.
 * [Counters](#Counters): Used to count how often something happens, for example, how often a certain button was pressed.
+* [Labeled counters](#Labeled-counters): Used to count how often something happens, for example which kind of crashes occured - `"uncaught_exception"` or `"native_code_crash"`.
 * [Timespans](#Timespans): Used to measure how much time is spent in a single task.
+* [Labeled timespans](#Labeled-timespans): Used to measure how much time is spent in a set of related tasks, for example different stages in a login process - the time taken for `"fill_form"`, `"auth_with_server"`, `"load_next_view"`.
 * [Timing Distributions](#Timing-Distributions): Used to record the distribution of multiple time measurements.
 * [Datetimes](#Datetimes): Used to record an absolute date and time, such as the time the user first ran the application.
 * [UUIDs](#UUIDs): Used to record universally unique identifiers (UUIDs), such as a client ID.
-* [Labeled Metrics](#Labeled-metrics): Used to record multiple metrics of the same type under different string labels, say every time you want to get a count of different error types in one metric.
+
+## Labeled metrics
+
+There are two types of metrics listed above - *labeled* and *unlabeled* metrics. If a metric is *labeled*, it means that for a single metric entry you define in `metrics.yaml`, you can record into multiple metrics under the same name, each identified by a different string label.
```

Comment:
I'd dig into the difference of the dynamic/static labels too, as shown in the [design doc](https://docs.google.com/document/d/186GI_2ljdqWqWTBk5P3QjHD3BwpMyd8kRayRAuz1-dk/edit#heading=h.z4j96kxogool)

Reply:
This section is just moved around in the doc.
I'll file a follow-up on that.